src/stylesheets/dsdl/core.css

```diff
@@ -6,10 +6,19 @@
 
 body {
   color: var(--dsdl-body-color);
-  font-family: var(--dsdl-font-stack);
+  font-family: var(--dsdl-body-font-stack);
   line-height: var(--line-height-normal);
 }
 
+h1, .h1,
+h2, .h2,
+h3, .h3,
+h4, .h4,
+h5, .h5,
+h6, .h6 {
+  font-family: var(--dsdl-heading-font-stack);
+}
```

Comment:
these exact same declarations are repeated below, yes?

Reply:
Oh yeah, you're right. I wrote it before adding the ones below and forgot to drop it.